src/com/biocollapse/model/Config.java

```diff
@@ -1,5 +1,148 @@
 package src.com.biocollapse.model;
 
 public class Config {
-    
+  private int infectionRadius;
+  private int infectionProbability;
+  private int incubationTime;
+  private int mortalityRate;
+  private int timeToDeath;
+  private int immunityChance;
+
+  private int hospitalCapacity;
+  private int isolationProbability;
+  private int hospitalProbability;
+  private int childrenRatio;
```

Comment:
I thought about the following because i came across it this morning: 

Do we need 3 Sliders for the 3 ratio Attributes or do we just preset it? Should the user be able to set the ratios himself?